src/Symfony/Component/DependencyInjection/Tests/Loader/YamlFileLoaderTest.php

```diff
@@ -137,6 +137,9 @@ public function testLegacyLoadServices()
         $this->assertEquals('getInstance', $services['factory_service']->getFactoryMethod());
     }
 
+    /**
+     * @group legacy
+     */
```

Comment:
I mean, I understand why, but as all assertions are not about deprecated features, you should probably extract the tests for deprecated features in another test.